### ISSUE TITLE
Fail on getting configuration value set to false

### ISSUE THREAD

**Issue by egodsk** [nestjs/nest-cli#526]
## I'm submitting a...

<pre><code>
[ ] Regression 
[x] Bug report?
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior

Looking at the code for `getValueOrDefault` in the `lib/compiler/helpers/get-value-or-default.ts` it returns `undefined` in the event that a value has been assigned the value `false` in the configuration

## Expected behavior

The value returned should be `false` when the configuration is assigned to `false`, not `undefined`

## Minimal reproduction of the problem with instructions

https://github.com/egodsk/nest-cli

I have added a test to my fork of the repo, which fails when trying to get a configuration which is false, as well as a possible fix. Interested in hearing your thoughts

## What is the motivation / use case for changing the behavior?

Having the ability to check if the value in the configuration was specifically set to `false` would be nice, instead of currently just returning false.

## Environment

```
[System Information]
OS Version     : Windows 10
NodeJS Version : v12.14.1

Failed to execute command: yarn --version
YARN Version    : Unknown
[Nest Information]
schematics version : 6.8.0
```

**Comment by egodsk** [comment]
UPDATE:
Found that if a project value was `false` as well, it would also return the wrong value. Have included a new commit in the forked repo with tests and a fix for the project specific configuration.

**Comment by egodsk** [comment]
I have created a PR #527 which includes the fixes as well as adds some extra features

**Comment by kamilmysliwiec** [comment]
Fixed in 6.14.0. Thank you @egodsk :)
